content/en/docs/setup/prep/deployment/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Deployment Best Practices
+description: General best practices for your Istio deployments.
+weight: 2
+icon: best-practices
+---
+
+We've identified the following general principles to help you get the most out
+of your Istio deployments.
+
+## Prefer simpler deployment models
+
+When choosing a [deployment model](/docs/setup/prep/deployment-models),
+prefer the **simplest** deployment possible. Keeping the complexity of your
+deployment to a minimum makes the task of managing your deployment easier.
+
+## Deploy clusters near your users
+
+Include clusters in your deployment across the globe for **geographic
+proximity to end-users**. The proximity helps your deployment have low latency.
+
+## Deploy across multiple availability zones
+
+Include clusters in your deployment **across multiple availability regions
+and zones** within each geographic region. This approach limits the size of the
+{{< gloss "failure domain" >}}failure domains{{< /gloss >}} of your deployment,
+and helps you avoid global failures.
+
+An Istio deployment running on Kubernetes satisfies the best practices
+above if it meets the following conditions:
+
+- The deployment runs a control plane per cluster.
```

Comment:
This doesn't actually follow for me. If you only have 1 cluster it fails to meet the goal while satisfying these conditions.

It also seems like the "control plane per cluster" is getting shoved in here and I'm not sure we actually want to recommend it.

Our best practice is to run a control plane per region, and to run in multiple regions.

What may help is if we add a best practice of only running a small number of large clusters, rather than a large number of small clusters.

If we do that, and suggest 1-2 clusters per region, then saying each cluster should have its own control plane is probably OK.

So maybe we should add two sections:
- A new section about preferring a small number of clusters per region, and using namespaces as tenancy.
- A finally "summary" section that puts all the other sections together with an example deployment model.